.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```

Comment:
It is not recommended to version IDE settings in a repository. Instead, you should git-ignore `.vscode` in your global Git settings.